orga/app/services/session.js

```diff
@@ -51,6 +51,16 @@ export default class CurrentSessionService extends SessionService {
     }
 
     const localeNotSupported = userLocale && !SUPPORTED_LANGUAGES.includes(userLocale);
+
+    if (localeNotSupported) {
+      this.data.localeNotSupported = true;
+
+      const localeNotSupportedBannerClosed = this.data.localeNotSupportedBannerClosed;
+
+      if (!localeNotSupportedBannerClosed) {
+        this.data.localeNotSupportedBannerClosed = false;
+      }
```

Comment:
Pas sure de comprendre ce qu'on veut faire ici (dur dur les conditions un vendredi soir) 🤯 
A quoi sert la `const localeNotSupportedBannerClosed` ? 
Est ce que faire un `this.data.localeNotSupportedBannerClosed = this.data.localeNotSupportedBannerClosed ?? false` ne revient pas au meme ?

Reply:
Discuter en mob pour la solution